src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -134,18 +136,27 @@ function drawHorizontalBarChartValues(
       const point = dataset.data[i];
 
       const yPosition = point.y;
-      let xPosition = value < 0 ? point.x + point.width / 2 : point.x - point.width / 2;
-      xPosition = Math.min(xPosition, xMax);
-      xPosition = Math.max(xPosition, xMin);
+      const textWidth = computeTextWidth(ctx, displayValue, { fontSize: 12 }, "px");
+      const barWidth = Math.abs(point.x - xZero);
```

Comment:
```suggestion
      const distanceFromAxisOrigin = Math.abs(point.x - xZero);
```
or something like that. "barWidth" is a bit confusing for stacked charts, we'd expect that it'd be the width of the bar, which is not the case